packages/plugins/rubygem-smart_proxy_m2/rubygem-smart_proxy_m2.spec

```
@@ -1,31 +1,41 @@
+# template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name smart_proxy_m2
 %global plugin_name m2
 
-%global foreman_proxy_min_version 1.20
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.0.1
-Release: 1%{?foremandist}%{?dist}
+Release: 3%{?foremandist}%{?dist}
 Summary: M2 smart proxy plugin for Foreman
 Group: Applications/Internet
 License: GPLv3
-URL: https://github.com/ianballou/smart_proxy_m2
+URL: http://github.com/ianballou/smart_proxy_m2
```

Fixed in master: https://github.com/ianballou/smart_proxy_m2/commit/fe4727addbec83c64f65f18407f648675c745257
```suggestion
URL: https://github.com/ianballou/smart_proxy_m2
```